### PR TITLE
Add conda to install instructions

### PR DESCRIPTION
**Pull request by marqh**
Add Conda instructions as primary installation mechanism

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
-latest release is available from https://github.com/SciTools/iris.
+You can either install Iris using the conda package manager or from source.
+
+Installing from conda
+---------------------
+
+Iris can be installed using the `conda <http://conda.pydata.org/>`_ package manager.
+
+Iris is available from conda for the following platforms:
+ * Linux 64-bit,
```
Comment (rhattersley) [inline INSTALL]: @pelson - Don't we have 32-bit linux as well?

Comment (niallrobinson) [inline INSTALL]: yup - I installed it at home

**Comment by pelson** [comment]
Although the tests are failing, I'd be happy to merge this. @rhattersley - any reason to hold back while the (unrelated) tests are failing?

**Comment by cpelley** [comment]
Looks like another natural earth data location issue! The data is missing off the github master source, however it is there under the v2.0 tag release.  We had a similar issue earlier last year when they changed the location of their data on their site.  This broke changed iris to use the updated location and fall-back to its github source if necessary, if I remember correctly.

File: INSTALL [hunk 2]
```diff
@@ -1,5 +1,31 @@
-Installing Iris currently requires you to build it from source. The 
-latest release is available from https://github.com/SciTools/iris.
+You can either install Iris using the conda package manager or from source.
+
+Installing from conda
+---------------------
+
+Iris can be installed using the `conda <http://conda.pydata.org/>`_ package manager.
```
Comment (rhattersley) [inline INSTALL]: Seems odd to have markup in plain text file? Most of the markup was removed [many moons ago](https://github.com/SciTools/iris/commit/9bdb292d84c8bab5b48168f47baac9b21a83fcb0#diff-1) so probably this should just be plain text too. (There are a few other bits of markup remaining which it would also be nice to remove, but that could be left to a separate PR.)

Comment (marqh) [inline INSTALL]: The reason for the markup is that this file is explicitly included in 
https://raw.githubusercontent.com/SciTools/iris/master/docs/iris/src/installing.rst
and built into the docs

I think it should stay to maintian this docs page

Comment (rhattersley) [inline INSTALL]: Thanks for pointing out the inclusion in the built docs - that's very helpful.

However, I still think we should avoid this markup:
 1. When used to create the built docs http[s] links are *already* converted to clickable links. There is no need to invoke markup. This is the style used in the vast majority of the rest of the file. You already have two other clickable links so perhaps the simplest change is just to remove this link.
 2. When viewed as a text file the markup just makes it harder to read.

File: INSTALL [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+Iris is available from conda for the following platforms:
+ * Linux 32-bit,
+ * Linux 64-bit,
+ * Mac OSX 64-bit, and
+ * Windows 32-bit and 64-bit.
+
+To install Iris from conda, you must first download and install conda, for example from
+http://conda.pydata.org/miniconda.html.
+
+Once conda is installed, you can install Iris from conda on any platform with the following command::
```
Comment (rhattersley) [inline INSTALL]: There are a number of long (i.e. >79) character lines here which make it hard to read as a text file.

File: INSTALL (hunk 2 above)
Comment (marqh) [inline INSTALL]: I see what you mean now.  the link conversion does work

i'll remove the link markup, it's not required

File: INSTALL (hunk 3 above)
Comment (marqh) [inline INSTALL]: no problem
i'll fix that

**Comment by marqh** [comment]
I removed some markup and fixed line lengths

I rebased in passing in the hope that this was now suitable to merge

**Comment by rhattersley** [comment]
Thanks @marqh

**Comment by rhattersley** [comment]
Grrrr :angry: ... I forgot this had already been done in #1503. Merge conflicts ahoy!

**Comment by esc24** [comment]
I'll have a go at a merging 1.7.x onto master unless anyone else is keen.

**Comment by rhattersley** [comment]
Thanks @esc24, but it's OK ... I'm already wrestling with it. :expressionless:

**Comment by esc24** [comment]
Phew.

**Comment by rhattersley** [comment]
See #1520.
